File: library/src/main/java/zjm/cst/dhu/library/inject/scope/PerContext.java
package zjm.cst.dhu.library.inject.scope;

import java.lang.annotation.Documented;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

import javax.inject.Qualifier;

/**
 * Created by zjm on 2017/5/9.
 */

@Qualifier
@Documented
@Retention(RetentionPolicy.RUNTIME)
public @interface PerContext {
    String value() default "Application";
}

File: library/src/main/kotlin/zjm/cst/dhu/library/inject/scope/PerContext.kt
package zjm.cst.dhu.library.inject.scope


import javax.inject.Qualifier

/**
 * Created by zjm on 2017/5/9.
 */

@Qualifier
@MustBeDocumented
@Retention(AnnotationRetention.RUNTIME)
annotation class PerContext(val value: String = "Application")
